pyslang/tests/test_cst_json.py

```diff
@@ -0,0 +1,91 @@
+# SPDX-FileCopyrightText: Michael Popoloski
+# SPDX-License-Identifier: MIT
+
+import pyslang
+
+
+def test_cst_json_basic():
+    """Test basic CST JSON serialization functionality."""
+    test_code = """
+    module test;
+        initial $display("Hello");
+    endmodule
+    """
+
+    tree = pyslang.SyntaxTree.fromText(test_code)
+
+    # Test default serialization (full mode)
+    json_str = tree.to_json()
+    assert isinstance(json_str, str)
+    assert "SyntaxTree" in json_str
```

Comment:
These tests are not very good. The output is deterministic, why not assert what the actual JSON string should be? Otherwise maybe the output is just garbage.